jsonnet/jsonnet.bzl

```diff
@@ -102,9 +108,17 @@ def _jsonnet_to_json_impl(ctx):
        "-J %s" % ctx.genfiles_dir.path,
        "-J %s" % ctx.bin_dir.path] +
       ["--var '%s'='%s'"
-       % (var, ctx.expand_make_variables("vars", jsonnet_vars[var],{})) for var in jsonnet_vars.keys()] +
+       % (var, ctx.expand_make_variables("vars", jsonnet_ext_strs[var],{})) for var in jsonnet_ext_strs.keys()] +
+      ["--var '%s'"
```

Comment:
Can you also change all instances of deprecated flags being sent to jsonnet? #47 will break tests due to deprecation warnings without it.

`WARNING: jsonnet eval -E, --var and --env are deprecated, please use -V or --ext-str`